src/pages/on-chain-governance/shared/localStorageHelper.ts

```diff
@@ -0,0 +1,32 @@
+import { RememberMasterNodeId } from "../enum/RememberMasterNodeId";
+
+// Function returns undefined when key is not found
+// when connection is provided function would check for undefined
+// and return "rememberMasternodeId"/"masternodeId" value from localStore
```

Comment:
this function is not only for `"rememberMasternodeId"/"masternodeId"`, it can be anything